lib/gui/components/drive-selector/controllers/drive-selector.js

```diff
@@ -153,6 +162,11 @@ module.exports = function(
     return shouldChangeDriveSelectionState(drive).then((canChangeDriveSelectionState) => {
       if (canChangeDriveSelectionState) {
         SelectionStateModel.setDrive(drive.device);
+
+        AnalyticsService.logEvent('Drive selected', {
+          method: 'double click'
```

Comment:
Only a minor point, but it looks a bit weird specifying `method: 'double click'` here when AFAICT there aren't any other 'Drive selected' events?